module/docs/entra-powershell-beta/Microsoft.Graph.Entra.Beta/Remove-EntraBetaApplicationPasswordCredential.md

```diff
@@ -12,38 +22,39 @@ Removes a password credential from an application.
 
 ## Syntax
 
-```
-Remove-EntraBetaApplicationPasswordCredential -ObjectId <String> -KeyId <String>
+```powershell
+Remove-EntraBetaApplicationPasswordCredential 
+ -ObjectId <String> 
+ -KeyId <String>
  [<CommonParameters>]
 ```
 
 ## Description
-The Remove-EntraBetaApplicationPasswordCredential cmdlet removes a password credential from an application in Azure Active Directory (AD).
+The `Remove-EntraBetaApplicationPasswordCredential` cmdlet removes a password credential from an application in Microsoft Entra ID.
 
 ## Examples
 
 ### Example 1: Remove an application password credential
-```
-PS C:\> $AppID = (Get-EntraBetaApplication -Top 1).ObjectId
-PS C:\> $KeyIDs = Get-EntraBetaApplicationPasswordCredential -ObjectId $AppId
-PS C:\> Remove-EntraBetaApplicationPasswordCredential -ObjectId $AppId -KeyId $KeyIds[0].KeyId
+```powershell
+$AppID = (Get-EntraApplication -Top 1).ObjectId
+$KeyIDs = Get-EntraBetaApplicationPasswordCredential -ObjectId $AppId
+Remove-EntraBetaApplicationPasswordCredential -ObjectId $AppId -KeyId $KeyIds[0].KeyId
```

Comment:
In example description we need t to add used parameters description for ObjectId and KeyId